fdbclient/Knobs.cpp

```diff
@@ -219,6 +219,10 @@ void ClientKnobs::initialize(bool randomize) {
 
 	init( CONSISTENCY_CHECK_RATE_LIMIT_MAX,        50e6 ); // Limit in per sec
 	init( CONSISTENCY_CHECK_ONE_ROUND_TARGET_COMPLETION_TIME,	7 * 24 * 60 * 60 ); // 7 days
+
+	// Large transaction
+	init( LARGE_TRANSACTION_CRITERIA,       1024 * 1024 );	// 1MB
```

Comment:
You probably want to BUGGIFY this down for testing?

Reply:
Let's discuss this via slack.